frontend/src/app/supportAdmin/DeviceType/DeviceForm.tsx

```diff
@@ -72,10 +110,103 @@ const DeviceForm = (props: Props) => {
             [],
           loincCode: device.loincCode,
           testLength: device.testLength ? device.testLength : 15,
+          supportedDiseaseTestPerformed,
         }
       : undefined;
   };
 
+  const createSupportedDiseaseRow = (index: number) => {
+    return (
+      <div className="grid-row grid-gap">
```

Comment:
Wondering if we should put each row in a `<fieldset>` to group them together since the inputs for that row is related to adding a supported disease and to make it clear which inputs/delete button are related to each supported disease? 🤔

Reply:
Visually, I don't think think there is a difference but it makes sense to do this!